0.8/elements/docs-menu.html

```diff
@@ -104,8 +104,19 @@
     </core-submenu>
 
     <core-submenu id="0.8" label="Polymer 0.8 (alpha)" icon="av:new-releases" class="guide">
-      <core-item label="Overview"><a href="/0.8/"></a></core-item>
-      <core-item label="API Documentation"><a href="/0.8/docs/api.html"></a></core-item>
+      <core-item label="About this release"><a href="/0.8/"></a></core-item>
+      <core-item label="Getting the code"><a href="/0.8/docs/start/getting-the-code.html"></a></core-item>      
+      <core-submenu id="0.8" label="Developer guide" icon="hardware:keyboard-arrow-down" class="guide">
+        <core-item label="Feature overview"><a href="/0.8/docs/devguide/feature-overview.html"></a></core-item>
+        <core-item label="Registration and lifecycle"><a href="/0.8/docs/devguide/registering-elements.html"></a></core-item>
+        <core-item label="Properties"><a href="/0.8/docs/devguide/properties.html"></a></core-item>
```

Comment:
For thee 0.8 links, you can use

`/{{versionPrefix}}/` instead of /0.8. Keep the 0.5 ones hardcoded